cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -268,6 +270,71 @@ def test_suspend(
 
 @pytest.mark.long_running
 @pytest.mark.uncollectif(lambda provider: provider.type != 'openstack')
+def test_pause(
+        setup_provider_funcscope, provider, testing_instance, soft_assert,
+        verify_vm_running):
+    """ Tests instance unpause
+
+    Metadata:
+        test_flag: power_control, provision
+    """
+    testing_instance.wait_for_vm_state_change(
+        desired_state=testing_instance.STATE_ON, timeout=720, from_details=True)
+    check_power_options(soft_assert, testing_instance, 'off')
+    testing_instance.power_control_from_cfme(
+        option=testing_instance.START, cancel=False, from_details=True)
+    flash.assert_message_contain("Pausing instance")
+    testing_instance.wait_for_vm_state_change(
+        desired_state=testing_instance.STATE_PAUSED, timeout=720,
+        from_details=True)
+    soft_assert(
```

Comment:
There's no point having a soft_assert at the end of a test, especially if there are no other soft_asserts. soft_asserts are for when there are multiple assertions that need to be made, but that we don't want to fail the test immediately if they don't assert.